Fix for crash in update_items() when trying to delete an item that isn't
in the dict.  (LP: #413387)

Index: a/lib/frontends/widgets/itemlist.py
===================================================================
--- a/lib/frontends/widgets/itemlist.py	2011-06-05 23:42:53.000000000 +0200
+++ b/lib/frontends/widgets/itemlist.py	2011-06-05 23:43:49.000000000 +0200
@@ -504,7 +504,10 @@
                 # Item not already displayed
                 if should_show:
                     to_add.append(info)
-                    del self._hidden_items[info.id]
+                    try:
+                        del self._hidden_items[info.id]
+                    except KeyError:
+                        pass # Item already gone
                 else:
                     self._hidden_items[info.id] = info
             else:
